langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/internal/api/MistralAiModerationResponse.java

```diff
@@ -0,0 +1,35 @@
+package dev.langchain4j.model.mistralai.internal.api;
+
+
+import java.util.List;
+
+public class MistralAiModerationResponse {
```

Comment:
please add `@JsonIgnoreProperties(ignoreUnknown = true)` to make sure we won't fail when Mistral API will start terurning more data

Reply:
done